BAZProject/Helpers/Constans.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Constans.swift
+//  BAZProject
+//
+//  Created by victor manzanero on 07/11/22.
+//
+
+import Foundation
+
+class Constans {
+    static let apiKey: String = "f6cd5c1a9e6c6b965fdcab0fa6ddd38a"
+}
+
+enum staticLabel {
```

Comment:
Los nombres de enumeraciones deben seguir el UpperCammelCase
```suggestion
enum StaticLabel {
```